src/customapplicationstyle.cpp

```diff
@@ -25,7 +25,7 @@ void CustomApplicationStyle::drawPrimitive(PrimitiveElement element, const QStyl
             painter->drawLine(option->rect.topLeft(), option->rect.topRight());
         } else {
             c.setAlpha(200);
-            QBrush brush(c);
+            brush.setColor(c);
```

Comment:
Here I would get rid of both `brush` variables instead. You can pass the `c` directly to `setBrush` or `fillRect` since colors implicitly convert to a solid brush.

Reply:
Perfect!